### ISSUE TITLE
Linter not running inside props function

### ISSUE THREAD

**Issue by caribou-code** [stylelint/stylelint#4283]
> Clearly describe the bug

Stylelint does not lint CSS code inside a styled components props arrow function.

> What CSS is needed to reproduce the bug?

```css
const Foo = styled.div`
  ${p => css`
    display: block;
    &::after {
      display ${p.isLoading ? 'block' : 'none'};
    }
  `};
`;
```

Note that in the above, the rule `rule-empty-line-before` should be triggered when runing the CLI stylelint command. The confirm this, the following code DOES work:

```css
const Foo = styled.div`
  display: block;

  &::after {
    display ${p => p.isLoading ? 'block' : 'none'};
  }
`;
```

I need to use the syntax in my first example for components that use the props object 30+ times (e.g. creating 1 function to access props instead of 30).

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "processors": [
    "stylelint-processor-styled-components"
  ],
  "extends": [
    "stylelint-config-standard",
    "stylelint-config-rational-order",
    "stylelint-config-styled-components"
  ]
}
```

> Which version of stylelint are you using?

`11.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint **/*.js

> What did you expect to happen?

A warning to be flagged for the `rule-empty-line-before`

> What actually happened (e.g. what warnings or errors did you get)?

No error thrown when the cli command is run

**Comment by evilebottnawi** [comment]
Please remove `stylelint-processor-styled-components`, it is deprecated, stylelint supports css-in-js out of box https://github.com/styled-components/stylelint-processor-styled-components/issues/278

**Comment by caribou-code** [comment]
Okay I think this is what was causing the issue. Thanks

**Comment by jeddy3** [comment]
Closing as processor issue.
